torch/distributed/rpc.py

```diff
@@ -74,39 +75,40 @@ def _init_rpc(backend=RpcBackend.PROCESS_GROUP,
             raise RuntimeError("self_rank argument {} doesn't match pg rank {}".format(
                                self_rank, group.rank()))
         # TODO: add try-except and destroy _agent in all processes if any fails.
-        _agent = ProcessGroupAgent(self_name, group, num_send_recv_threads)
-        init_rref_context(_agent)
+        _agent = ProcessGroupAgent(self_name, group)
```

Comment:
Is this intentional? Perhaps there was a race with landing `num_send_recv_threads` change?

Reply:
ahh, good catch, this was NOT intentional. My rebase messed it up. Let me update.